### PR TITLE
Chore/supporter section 85

### PR DESCRIPTION
**Pull request by stylesuxx**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,12 +45,34 @@ You can also do it manually by downloading the [release binaries](https://github
...
+
+* 406FPV - for donating a F722 based flight-controller, motors and a whole 1105 based micro quad
+* belrik - for donating a set of 2207 motors
+* Ced_51 - for dontaing multiple sets of different motors (2207, 2306)
+* geordil - for donating a F722 based flight-controller
+* Miguel Juncal - for donating a 5" quad and a set of Sunnysky motors
+* RabbitAmbulance - for donating a set of 2207 motors
+
+### Manufacturers
+We are in a very lucky position to have a couple of manufacturers who keep helping us out continously and who deserve a shoutout:
```
Comment (damosvil) [inline README.md:69]: No longer a couple ;)

Comment (stylesuxx) [reply README.md:69]: Ahhh, in the grand scheme of things it's still a couple :-D if we cross the threshold of 50% I will adapt it to "a significant amount".

How about:
> We are in a very lucky position to have manufacturers who keep helping us out continuously and who deserve a shoutout:

Comment (damosvil) [reply README.md:69]: I think that it sounds good

Comment (stylesuxx) [reply README.md]: Fixed.

**Review by damosvil — APPROVED** [review]

**Pull request merged**